### PR TITLE
clean up some sonarlint warnings

### PR DESCRIPTION
**Pull request by RangerRick**

### CODE REVIEW

File: config/src/main/java/org/opennms/integration/api/xml/schema/thresholding/Service.java [hunk 1]
```diff
@@ -149,20 +148,22 @@ public int hashCode() {
...
-    public boolean equals(final Object obj) {
+    public boolean equals(Object obj) {
         if ( this == obj ) {
             return true;
         }
-
-        if (obj instanceof Service) {
-            final Service that = (Service)obj;
-            return Objects.equals(this.m_name, that.m_name)
-                    && Objects.equals(that.m_interval, that.m_interval)
```
Comment (RangerRick) [inline config/src/main/java/org/opennms/integration/api/xml/schema/thresholding/Service.java:160]: oooops! lol

File: config/src/main/java/org/opennms/integration/api/xml/schema/eventconf/EventLabelComparator.java [hunk 2]
```diff
@@ -32,11 +32,11 @@
 import java.util.Comparator;
 
 public class EventLabelComparator implements Comparator<Event>, Serializable {
-    private static final long serialVersionUID = 7976730920523203921L;
+    private static final long serialVersionUID = 1L;
 
     @Override
     public int compare(final Event e1, final Event e2) {
-        if (e1.getEventLabel() == e2.getEventLabel()) return 0;
+        if (e1.getEventLabel().equals(e2.getEventLabel())) return 0;
```
Comment (synqotik) [inline config/src/main/java/org/opennms/integration/api/xml/schema/eventconf/EventLabelComparator.java:39]: The null checks should be placed before the `equals` check.

**Review by synqotik — CHANGES_REQUESTED** [review]
Looks good except for one comment which should be addressed.

**Review by synqotik — APPROVED** [review]
Looks good!

**Review by christianpape — APPROVED** [review]
LGTM.

File: config/src/main/java/org/opennms/integration/api/xml/schema/eventconf/EventLabelComparator.java (hunk 2 above)
Comment (synqotik) [reply config/src/main/java/org/opennms/integration/api/xml/schema/eventconf/EventLabelComparator.java]: Looks like Sonar caught another issue which needs to be fixed ... if only Java had an optional chaining operator...

Comment (RangerRick) [reply config/src/main/java/org/opennms/integration/api/xml/schema/eventconf/EventLabelComparator.java]: It sure did! Ergh. Good for you, Sonar.

**Pull request merged**